setup.py

```diff
@@ -17,7 +17,7 @@
     package_dir={'moztelemetry': 'moztelemetry'},
     install_requires=['boto', 'boto3', 'ujson', 'requests', 'protobuf',
                       'expiringdict', 'functools32', 'futures', 'py4j',
-                      'pandas>=0.14.1', 'numpy>=1.8.2', 'findspark',
+                      'pandas>=0.14.1', 'numpy>=1.8.2', 'pyspark',
```

Comment:
Hmm, is this actually required? seems to me the tests would add this due to `environment.yaml`.

Reply:
Let's find out :)